notebooks/official/datasets/get_started_bq_datasets.ipynb

```diff
@@ -103,7 +109,7 @@
         "When doing E2E MLOps on Google Cloud, following are the best practices when dealing with structured (tabular) data in BigQuery:\n",
         "\n",
         "- For AutoML training:\n",
-        "  - Create a managed dataset with Vertex AI `TabularDataset`.\n",
+        "  - Create a managed dataset with Vertex AI TabularDataset.\n",
```

Comment:
- change: "Vertex AI TabularDataset"
- to: Vertex AI tabular dataset"

For reference, see: https://cloud.google.com/vertex-ai/docs/tutorials/tabular-bq-prediction/create-dataset

Reply:
Question: Here `TabularDataset` is a python class. Shouls that still be changed?